DigitalLearningSolutions.Web/Views/ForgotPassword/ForgotPassword.cshtml

```diff
@@ -0,0 +1,20 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.ForgotPassword
+@model ForgotPasswordViewModel
+@{
+  ViewData["Title"] = "Digital Learning Solutions - Reset Your Password";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 id="page-heading">Reset your password</h1>
+
+    <form method="post" role="form" asp-action="SubmitEmail">
+      <label class="nhsuk-label nhsuk-u-margin-bottom-5" for="emailAddress">
+        Enter your email address and we will send you password reset instructions.
+      </label>
+      <input class="nhsuk-input nhsuk-u-margin-bottom-5" type="email" id="emailAddress" required aria-required="true" asp-for="EmailAddress" value="@Model.EmailAddress"/>
+      <input class="nhsuk-button" type="submit" value="Reset" />
```

Comment:
I think we usually use tags in the forms as they should be more flexible than input type="submit"